test/acceptance/features/bindAppToService.feature

```diff
@@ -585,3 +585,33 @@ Feature: Bind an application to a service
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "binding-request-backend-a" should be changed to "False"
         And Secret "binding-request-backend-a" contains "BACKEND_READY" key with value "true"
         And Secret "binding-request-backend-a" contains "BACKEND_HOST" key with value "example.common"
+
+        @negative
+        Scenario: Create Service Binding with empty services
+        When Invalid Service Binding is applied
+            """
+            apiVersion: operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: binding-request-empty-services
+            spec:
+                services:
+            """
+        Then Openshift throws an error "invalid: spec.services: Invalid value: \"null\""
```

Comment:
Let be more generic here, for future plain k8s support: how about "Cluster reports the error "invalid: spec.services: Invalid value: \"null\""

Also, we really need to check that the submitted SB instance is not persisted in the cluster

Reply:
@pedjak We can be even more generic here, see https://github.com/redhat-developer/service-binding-operator/pull/698#discussion_r498217503

Reply:
+1 on checking SB is not persisted.

Reply:
I made some changes PTAL.